### PR TITLE
Update It

### PR DESCRIPTION
**Pull request by shockz09**
Updated and added the link of the exact web design course of freecodecamp so that newbies have no problem navigating through it!.

 - [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).

#### 1.Describe the changes made:

...your text here

#### 2. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

#XXXXX

### CODE REVIEW

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]
I like the change of linking directly to the responsive web design section, but I think we need to change the instructions we're giving to students as well to match this change. Maybe something along the lines of "do only the courses/lessons in the linked section, do not do any of other sections/certifications such as `Javascript Algorithms`" though I'm not married to that specific wording.

**Comment by shockz09** [comment]
Thanks for you suggestion for changes. I forgot that to write. I am going to write that the do this course only

File: foundations/the_front_end/html_css_basics.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Look through these now and then use them to test yourself after doing the assign
...
+  1. Dive right into learning HTML and CSS with [freeCodeCamp's interactive tutorials](https://www.freecodecamp.org/learn/responsive-web-design/) to introduce you to these new concepts. Do only the courses/lessons in the linked section, do not do any of other sections/certifications such as Javascript Algorithms". It's long and, if you're new, might take you at least a few days to get through. **This resource is meant to help you develop familiarity, so do not expect to memorize everything.** Instead, expect to google these concepts in the future when you need to use them. They also have some projects at the end of this section. They're good practice, but you can consider them optional. (A large part of their proposed '300 hours' for completion comes from these projects, so it probably won't take you that long if you only do the exercises).
```
Comment (wise-king-sullyman) [inline foundations/the_front_end/html_css_basics.md:32]: `do not do any of other sections` -> `do not do any of the other sections`

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]
It needs one last small wording change that I see, then it'll be approval time!

**Comment by shockz09** [comment]
Sir , I have done the things you requested

**Review by wise-king-sullyman — APPROVED** [review]
Looks perfect! Thank you for your contribution!

**Comment by shockz09** [comment]
Welcome, Sir. Like this only I want to contribute to open-source projects

**Pull request merged**